tests/Test/AbstractFixerTestCase.php

```diff
@@ -143,7 +143,7 @@ protected function tearDown()
             $config = $sample->getConfiguration();
             if (null !== $config) {
                 static::assertTrue($fixerIsConfigurable, sprintf('[%s] Sample #%d has configuration, but the fixer is not configurable.', $fixerName, $sampleCounter));
-                static::assertInternalType('array', $config, sprintf('[%s] Sample #%d configuration must be an array or null.', $fixerName, $sampleCounter));
+                static::assertIsArray($config, sprintf('[%s] Sample #%d configuration must be an array or null.', $fixerName, $sampleCounter));
```

Comment:
just to ensure - this change is done manually or automatically by one of the PHP CS Fixer rule ?

Reply:
This is done by `php_unit_dedicate_assert_internal_type`